editor.planx.uk/src/theme.ts

```diff
@@ -212,6 +212,10 @@ const getThemeOptions = (primaryColor: string): ThemeOptions => {
           sizeLarge: {
             fontSize: "1.188rem",
             fontWeight: FONT_WEIGHT_SEMI_BOLD,
+            width: "100%",
+            "@media (min-width: 768px)": {
+              width: "auto",
+            },
```

Comment:
Similar to above, we may be best referring to the breakpoint directly here. In order to do this, we need to create a `Breakpoints` object first and use this as part of the theme - this is very similar to how the palette object is handled. If the below is not clear we can catch up on Slack tomorrow about it 👌 

_Inside getThemeOptions()_
```ts
const palette = createPalette(deepmerge(DEFAULT_PALETTE, teamPalette));
const breakpoints = createBreakpoints({
  values: {
    xs: 0,
    sm: 500,
    md: 768, // Used with Container as general max-width
    lg: 1280,
    xl: 1920,
  },
});

const themeOptions: ThemeOptions = {
  // Set default spacing unit to match GOV.UK
  spacing: 10,
  typography: {
    ...
  }
  palette,
  breakpoints,
  ...
  components: {
    ...
    MuiButton: {
      sizeLarge: {
        fontSize: "1.188rem",
        fontWeight: FONT_WEIGHT_SEMI_BOLD,
        [breakpoints.up("md")]: {
          width: "auto",
        },
        width: "100%",
      },
    },
  }
}

Reply:
Also - this is such a quality change on mobile - makes a huge difference....! No more sore thumb stretching 😂